Dockerfile

```diff
@@ -26,10 +26,16 @@ FROM registry.access.redhat.com/ubi9/ubi-minimal:9.3
 COPY --from=builder /opt/app-root/src/manager /
 USER 65532:65532
 
+# Required for ecosystem-cert-preflight-checks
+# https://access.redhat.com/documentation/en-us/red_hat_software_certification/2024/html-single/red_hat_openshift_software_certification_policy_guide/index#assembly-requirements-for-container-images_openshift-sw-cert-policy-introduction
+COPY LICENSE /licenses/LICENSE
+
+LABEL name="build-service"
 LABEL description="RHTAP Build Service operator"
 LABEL io.k8s.description="RHTAP Build Service operator"
 LABEL io.k8s.display-name="build-service-operator"
 LABEL io.openshift.tags="rhtap"
 LABEL summary="RHTAP Build Service"
+LABEL com.redhat.component="build-service-operator"
```

Comment:
1. RHTAP -> Konflux.
2. In `com.redhat.component` I'd mention Konflux too (unless we 100% sure it will be combined with other items that already has mention for Konflux).

Reply:
:+1: updated